src/main/java/frc/robot/subsystems/Swerve/SwerveSubsystem.java

```diff
@@ -0,0 +1,156 @@
+package frc.robot.subsystems.Swerve;
+
+import frc.robot.Constants;
+import frc.robot.RobotMap;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.kinematics.SwerveModulePosition;
+
+import com.ctre.phoenix.sensors.Pigeon2;
+
+import edu.wpi.first.math.controller.HolonomicDriveController;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.estimator.SwerveDrivePoseEstimator;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class SwerveSubsystem extends SubsystemBase {
```

Comment:
I would just refactor this to `DrivebaseSubsystem` we don't need to differentiate with "Swerve" here.

Reply:
After consideration, I agree.